app/rfid.py

```diff
@@ -11,26 +11,24 @@
 
 
 class sensor:
-    def __init__(self):
+    def __init__(self, rfid_record_time):
         global ser
         ser = serial.Serial(
             port="/dev/ttyAMA0",
             baudrate=9600,
             parity=serial.PARITY_NONE,
             stopbits=serial.STOPBITS_ONE,
             bytesize=serial.EIGHTBITS,
-            timeout=10,
+            timeout=rfid_record_time,
         )
 
-    def read(self, rfid_record_time):
-        time_start = time.time() # record time
-        while (time.time() - time_start) < record_time # record for 120 secs unless read
-            ser.reset_input_buffer() #  clean buffer
-            ser.reset_output_buffer() # clean buffer
-            ser.write(b"sd2\r\n") # set mode of rfid
-            a = ser.read_until(size=19).decode(
-            "utf-8"
-            )  # 16 byte  + \r + \n somehow is 19 not 18
-            if len(a) > 15: # if read then return out of function
-                return a # return and break out of functions
+    def read(self):
+        ser.reset_input_buffer() #  clean buffer
```

Comment:
The comment `# record for fixed time after pir reading` plus the fact there's a variable `rfid_record_time` set to 120 implies that something will record for 2 mins.

However, this function doesn't use that variable at all. Is that deliberate? If so, we should remove all reference to the variable here and in the calling function.